common-test/src/main/java/io/strimzi/test/k8s/BaseKubeClient.java

```diff
@@ -287,6 +288,20 @@ public K waitForResourceDeletion(String resourceType, String resourceName) {
         return (K) this;
     }
 
+    public K waitForResourceUpdate(String resourceType, String resourceName, String startTime) {
+
+        TestUtils.waitFor(resourceType + " " + resourceName + " update",
+                1_000L, 240_000L, () -> {
+                try {
+                    return !startTime.equals(JsonPath.parse(getResourceAsJson(resourceType, resourceName)).
```

Comment:
Can we check that startTime before the update is less than startTime after the update? I already describe this case in a comment for method `getResourceCreateTimestamp()` in this PR.